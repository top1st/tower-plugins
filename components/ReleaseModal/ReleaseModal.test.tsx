import { shallow } from 'enzyme';
import * as React from 'react';
import { ReleaseModal } from './';

const defaultProps = {
    modalClose: jest.fn(),
    open: false,
    handleClick: jest.fn(),
    handleChangeSelect: jest.fn(),
    amount: '',
};

describe('Release Modal page test', () => {
    it('should render', () => {
        const wrapper = shallow(<ReleaseModal {...defaultProps} />);
        expect(wrapper).toMatchSnapshot();
        expect(wrapper).toBeDefined();
    });
});
