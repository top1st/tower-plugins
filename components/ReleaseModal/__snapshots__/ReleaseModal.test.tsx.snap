// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Release Modal page test should render 1`] = `ShallowWrapper {}`;
